routers/api/v1/misc/version.go

```diff
@@ -0,0 +1,15 @@
+// Copyright 2017 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package misc
+
+import (
+	"code.gitea.io/gitea/modules/context"
+	"code.gitea.io/gitea/modules/setting"
+)
+
+// ServerVersion shows the version of the Gitea server
+func ServerVersion(ctx *context.APIContext) {
+	ctx.Write([]byte(setting.AppVer))
```

Comment:
There exists an API named `Version` already (https://github.com/go-gitea/gitea/blob/master/vendor/code.gitea.io/sdk/gitea/gitea.go#L17) for 'library's version'.

I am afraid that it would lead to confusion if I have the both called 'Version'.